autonomi/src/client/data_types/pointer.rs

```diff
@@ -96,10 +101,10 @@ impl Client {
         debug!("Paying for pointer at address: {address:?}");
         let (payment_proofs, _skipped_payments) = self
             // TODO: define Pointer default size for pricing
-            .pay(
-                DataTypes::Pointer.get_index(),
-                std::iter::once((xor_name, 128)),
-                wallet,
+            .pay_for_content_addrs(
+                DataTypes::Pointer,
+                std::iter::once((xor_name, size_of::<Pointer>())),
```

Comment:
`size_of::<Pointer>()` is the in-memory (Rust) size. Is that the same as the size of the data we store (serialized)? Or doesn't this need to be that accurately calculated/specified?

Reply:
That's an excellent question, the Pointer size is fixed though so this field is useless for pointers, I was in favour of completely removing it

Reply:
For now, made it a universally accessible information: `Pointer::size()` that we can decide to change down the line.